edxval/views.py

```diff
@@ -75,7 +77,7 @@ class VideoList(generics.ListCreateAPIView):
     """
     GETs or POST video objects
     """
-    authentication_classes = (OAuth2Authentication, SessionAuthentication)
+    authentication_classes = (JwtAuthentication, OAuth2Authentication, SessionAuthentication)
```

Comment:
Why is the `OAuth2Authentication` being left in this list? Isn't `JwtAuthentication` sufficient?

Reply:
@doctoryes I mistakenly thought it was still required for the tests to pass, but it turns out that they only need SessionAuthentication to be there.  So I have now removed OAuth2Authentication.